src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -566,7 +566,7 @@ public void updateFeedbackQuestionNumber(FeedbackQuestionAttributes newQuestion)
         }
         
         adjustQuestionNumbers(oldQuestionNumber, newQuestionNumber, questions);
-        updateFeedbackQuestion(newQuestion);
+        updateFeedbackQuestionWithResponseRateCheck(newQuestion);
```

Comment:
Ok, let's do that in another issue then, since it seems it will affect multiple places in the code